### ISSUE TITLE
Support selecting packages

### ISSUE THREAD

**Issue by cptartur** [software-mansion/protostar#1998]
Current CLI version has hardcoded path to an example package. Instead, some package selection mechanism should be implemented.

Ideas: 
1. Detect the current working directory and search for packages there (`MetadataCommand` should be able to do that automatically)
2. Provide some sort of CLI mechanism for choosing a package

**Comment by MaksymilianDemitraszek** [comment]
Shouldn't this be a responsibility of Scarb?

**Comment by cptartur** [comment]
I don't think this can be easily handled in Scarb. Scarb metadata rust package will find all packages within a workspace you are currently in. 

Filtering which packages to run tests on has to be done on our side. Metadata even added an ability to easily filter packages specifically for this purpose.

**Comment by MaksymilianDemitraszek** [comment]
Blocked by scarb workspaces
